.buildkite/pull-requests.json

```diff
@@ -31,22 +31,6 @@
             "skip_target_branches": [ ],
             "skip_ci_on_only_changed": ["^x-pack/elastic-agent/README.md", "^x-pack/elastic-agent/docs/.*", "^x-pack/elastic-agent/devtools/.*" ],
             "always_require_ci_on_changed": ["^x-pack/elastic-agent/.*", ".buildkite/x-pack/elastic-agent/.*", "^go.mod", "^pytest.ini", "^dev-tools/.*", "^libbeat/.*", "^testing/.*"]
-        },
-        {
-            "enabled": true,
-            "pipelineSlug": "beats-winlogbeat",
```

Comment:
https://github.com/elastic/beats/pull/39154 is merged, how this is showing on the PR?

Reply:
I think I need to rebase my branch. But after that we won't be able to test packaging